java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -685,6 +662,10 @@ class ReferenceModeStore private constructor(
         })
     }
 
+    suspend fun close() {
+        backingStore.clearStoresCache()
+    }
```

Comment:
Should we lift the `close` method to the `Store`/`ActiveStore` interface?

Reply:
Yes!